examples/ui/client-capabilities/interaction/text-field.js

```diff
@@ -0,0 +1,8 @@
+import { CodeLabel, CodeSection } from '../../documentationBase';
+import { registerComponents } from '../../rapidImports';
+
+registerComponents();
+
+export default function TextFieldDemo({ children, color }) {
+	return (<CodeSection><rapid-text-field placeholder="Input text here">Sample Text Field</rapid-text-field></CodeSection >)
```

Comment:
can we have one each of 
type
{"email", "password","tel","number","text","url"}   

One 'start' slot with a `$` sign
One 'end' slot with a '%' sign